src/vuestic-theme/vuestic-directives/Layout.js

```diff
@@ -4,13 +4,13 @@ export default {
     let navbar = el.querySelector('.vuestic-navbar')
     let sidebar = el.querySelector('.vuestic-sidebar')
 
-    const desktopWidth = 992
+    const mobileWidth = 415
```

Comment:
why 415?

Reply:
it is a maximum mobile width (i check it from all mobile devices in browser), that helps us to divide sidebar behaviour between mobile and desktop version of app, related issue:
https://github.com/epicmaxco/vuestic-admin/issues/283

Reply:
ok, please check with @smartapant